packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -20,27 +19,39 @@ import {
     DetachedField,
     AnchorSet,
     detachedFieldAsKey,
-    keyAsDetachedField,
     Value,
     Delta,
-    JsonableTree,
-    getGenericTreeField,
     UpPath,
     Anchor,
     visitDelta,
     ITreeCursorNew,
+    CursorLocationType,
+    TreeSchemaIdentifier,
+    TreeValue,
+    MapTree,
+    getMapTreeField,
+    keyAsDetachedField,
+    rootFieldKeySymbol,
 } from "../../core";
 import { brand, fail } from "../../util";
-import { jsonableTreeFromCursor } from "../treeTextCursor";
+import { CursorWithNode, SynchronousCursor } from "../treeCursorUtils";
+import { mapTreeFromCursor, singleMapTreeCursor } from "../mapTreeCursor";
+
+function makeRoot(): MapTree {
+    return {
+        type: brand("above root placeholder"),
+        fields: new Map(),
+    };
+}
 
 export class ObjectForest extends SimpleDependee implements IEditableForest {
```

Comment:
Needs a doc comment

Reply:
Done. Annoying this is mostly rehashing the folder level readme which we can't easily link from here.